deployments/mainnet/eth/deploy.ts

```diff
@@ -0,0 +1,45 @@
+import { Deployed, DeploymentManager } from '../../../plugins/deployment_manager';
+import { DeploySpec, deployComet } from '../../../src/deploy';
+import { getConfiguration, NetworkConfiguration } from '../../../src/deploy/NetworkConfiguration';
+
+export default async function deploy(deploymentManager: DeploymentManager, deploySpec: DeploySpec): Promise<Deployed> {
+  // XXX configuration params for assets are just placeholders
+
+  // Deploy WstETHPriceFeed
+  const wstETHPriceFeed = await deploymentManager.deploy(
+    'wstETHPriceFeed',
+    'WstETHPriceFeed.sol',
+    [
+      '0xcfe54b5cd566ab89272946f602d76ea879cab4a8', // stETHtoUSDPriceFeed
+      '0x7f39c581f595b53c5cb19bd0b3f8da6c935e2ca0'  // wstETH
+    ]
+  );
+
+  const config = await deploymentManager.readConfig<NetworkConfiguration>();
+  const wstETHaddress = config.assets.wstETH.address;
+
+  let { assetConfigs } = await getConfiguration(deploymentManager);
+  assetConfigs = (assetConfigs || []).map(ac => {
+    return ac.asset === wstETHaddress ? { ...ac, ...{ priceFeed: wstETHPriceFeed.address } } : ac;
+  });
+
+  // Deploy all Comet-related contracts
+  const deployed = await deployComet(
+    deploymentManager,
+    deploySpec,
+    { assetConfigs }
```

Comment:
Something seems off here, we shouldn't have to do all this... 🤔

Reply:
The additional setup is so that we can deploy the `wstETHPriceFeed` contract and then pass it in as an argument to `deployComet`, overriding the value from `configuration.json`.

We normally don't have to do this since all of our price feeds are already deployed.

One other option would be to deploy the price feed manually in advance, and just add that address to the configuration file.

Reply:
I think we might be able to get rid of everything except the deploy if we use ```getContractAddress(`${assetName}:priceFeed`, assetConfig.priceFeed)``` here https://github.com/compound-finance/comet/blob/main/src/deploy/NetworkConfiguration.ts#L98 i.e. with the current one as the fallback (and I guess here too for parity https://github.com/compound-finance/comet/blob/main/src/deploy/NetworkConfiguration.ts#L134)

Reply:
that's awesome. made the update